### PR TITLE
fix(subsite): Fix logo size on subsite section

### PR DESCRIPTION
**Pull request by kreafox**

### CODE REVIEW

**Review by tiberiuichim — APPROVED** [review]

File: theme/themes/eea/extras/header.less [hunk 1]
```diff
@@ -298,27 +298,33 @@
 
 /* Subsite */
 .eea.header .logo-wrapper {
+  position: relative;
   display: flex;
   align-items: center;
-  gap: 2em;
+  margin-top: @mobileLogoMarginTop;
+  gap: 30px;
```
Comment (ichim-david) [inline theme/themes/eea/extras/header.less:305]: @kreafox  please add a variable and reference it

File: theme/themes/eea/extras/header.less [hunk 2]
```diff
@@ -298,27 +298,33 @@
...
+  font-size: @mobileSubsiteFontSize;
   font-weight: @subsiteFontWeight;
-  color: @subsiteColor;
-  position: relative;
+  line-height: @subsiteLineHeight;
 
   &:before {
     position: absolute;
+    top: 50%;
+    left: -15px;
```
Comment (ichim-david) [inline theme/themes/eea/extras/header.less:323]: @kreafox  also here these values might need to be changed in another thematic website

File: theme/themes/eea/extras/header.less [hunk 3]
```diff
@@ -545,6 +555,15 @@
 }
 
 @media only screen and (min-width: @computerBreakpoint) {
+  .eea.header .subsite-logo {
+    position: absolute;
+    z-index: 1;
+    top: 50%;
+    left: calc(100% + 30px);
```
Comment (ichim-david) [inline theme/themes/eea/extras/header.less:562]: @kreafox 30px should be a variable I think that it's like the gap value
so I would add that as a variable and reference it here

**Review by ichim-david — CHANGES_REQUESTED** [review]
@kreafox I've left some comments on some things that I think you need to add as variable definitions and perhaps add a comment when you need to override on tablet size what it fixes

**Review by ichim-david — APPROVED** [review]

**Pull request merged**